playbooks/bootstrap.yml

```diff
@@ -1,6 +1,35 @@
 ---
-- import_playbook: "./bootstrap_tower.yml"
-- import_playbook: "./bootstrap_integreatly.yml"
-- import_playbook: "./bootstrap_cluster_create.yml"
-- import_playbook: "./bootstrap_cluster_teardown.yml"
-- import_playbook: "./bootstrap_validation.yml"
\ No newline at end of file
+- hosts: localhost
+  gather_facts: no
+  tasks:
+    - debug:
+        msg: "ERROR: No branch/tag of the configuration repo defined defaulting to master"
+      when: integreatly_version is not defined
+
+    - set_fact:
+        tower_configuration_project_scm_branch: "master"
+        tower_credentials_project_scm_branch: "master"
+        integreatly_version: "master"
+      when: integreatly_version is not defined
+
+    - set_fact:
+        bootstrap: "all"
+      when: bootstrap is not defined
+
+#
+# all
+#
+- import_playbook: "./bootstrap_all_jobs.yml"
+  when: bootstrap == "all"
```

Comment:
Defaulting the `bootstrap` variable to all would be fine. We can just set it in the defaults/main.yml of the role rather than overriding it here.

Reply:
Actually, maybe if we default to `poc` instead